Formula/libbpf.rb

```diff
@@ -0,0 +1,36 @@
+class Libbpf < Formula
+  desc "Automated upstream mirror for libbpf stand-alone build"
+  homepage "https://github.com/libbpf/libbpf"
+  url "https://github.com/libbpf/libbpf/archive/refs/tags/v0.8.1.tar.gz"
+  sha256 "7bda8187efc619d1eb20a1ba5ab949dd68d40dd44945310c91ac0f915fa4a42b"
+  license "BSD-2-Clause"
+
+  depends_on "elfutils"
+  depends_on :linux
+  depends_on "linux-headers@5.16"
+  depends_on "pkg-config"
+  depends_on "zlib"
+
+  def install
+    chdir "src" do
+      system "make"
+      system "make", "install", "PREFIX=#{prefix}"
+    end
+  end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      #include <bpf/bpf.h>
+      #include <stdio.h>
+
+      int main() {
+
+        printf("%d.%d", LIBBPF_MAJOR_VERSION, LIBBPF_MINOR_VERSION);
+        return(0);
+      }
+    EOS
+    system ENV.cc, "test.c", "-I#{include}", "-L#{lib}", "-lbpf",
```

Comment:
This might also need `-I#{Formula["linux-headers@5.16"].opt_include}` as an include flag so that it can find the headers.

Reply:
Thanks it looked like this worked. The test was only referencing macros before (so didn't need to link against the library), but I'm updating it to call a function which will require being linked.